tgui/packages/tgui/interfaces/Pacman.js

```diff
@@ -0,0 +1,174 @@
+import { useBackend } from "../backend";
+import { Button, Box, Section, ProgressBar, LabeledList, Knob, NumberInput, Flex, Grid } from "../components";
+import { Window } from "../layouts";
+
+export const Pacman = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    broken,
+    anchored,
+    active,
+    fuel_type,
+    fuel_usage,
+    fuel_stored, // current fuel level
+    fuel_cap, // fuel_capacity
+    is_ai,
+    tmp_current, // temperature_current
+    tmp_max, // temperature_max
+    tmp_overheat, // temperature_overheat
+    output_max,
+    power_gen,
+    output_set,
+  } = data;
+  let fuelRatio = fuel_stored / fuel_cap;
+  let tmpRatio = tmp_current / tmp_max; // temperature ratio
+  let output_watts = output_set * power_gen;
+  let fuel_sec = Math.round(fuel_stored / fuel_usage);
+  let fuel_min = Math.round(fuel_sec / 60);
+  let usage = fuel_sec > 120
+    ? `${fuel_min} minutes`
+    : `${fuel_sec} seconds`;
+  return (
+    <Window>
+      <Window.Content>
+        {((broken || !anchored) && (
+          <Section title="Status">
+            {!!broken && (
+              <Box color="orange">
+                The generator is malfunctioning.
```

Comment:
```suggestion
                The generator is malfunctioning!
```
Its a bad thing, so I feel an `!` is warranted

Reply:
Done